package.json

```diff
@@ -17,7 +17,7 @@
 		"url": "https://github.com/Automattic/jetpack/issues"
 	},
 	"scripts": {
-		"watch": "yarn install-if-deps-outdated && yarn gulp watch",
+		"watch": "yarn build && yarn gulp watch",
```

Comment:
Observation: We're likely doing a bit of extra work here by running the extensions build twice. I'd expect extensions watch task to build it, so the first build could be `build-client`. The tradeoff is that someone needs to keep that in mind when working on the build and it's the type of thing that tends to slip through the cracks, good to leave this as is.